src/options/options.html

```diff
@@ -81,8 +81,11 @@ <h6 class="margin-top-medium vocab heading">All Providers</h6>
           </div>
         </div>
         <div class="content" data-content-no="2">
-          <!-- Persist search even when extension is closed?
-          yes (checkbox) -->
+          <h6 class="vocab heading">Options</h6>
```

Comment:
The template should be:
(heading) Load last search result when you open the extension?
(checkbox) Yes

Reply:
Actually I wrote the code for implementing whether the user wants to store search or not!